wgpu/src/lib.rs

```diff
@@ -5428,55 +5035,25 @@ pub enum Error {
     /// Out of memory error
     OutOfMemory {
         /// Lower level source of the error.
-        #[cfg(any(
-            not(target_arch = "wasm32"),
-            all(
-                feature = "fragile-send-sync-non-atomic-wasm",
-                not(target_feature = "atomics")
-            )
-        ))]
+        #[cfg(send_sync)]
```

Comment:
This are also visible, which makes sense:
![image](https://github.com/gfx-rs/wgpu/assets/1645124/4177cc4b-960f-455b-a7be-75e01181ff09)
Would also recommend to remove that.

Reply:
I know of at least one project that uses this, it can be super useful if you also depend on wgpu-core. So removing isn't an option.
These small wrinkles in the generated docs are super annoying but at least most of it happens only on fairly advanced stuff, so I'm not _too_ worried

Reply:
Sorry, I was using the word "remove" in context of what I was proposing in https://github.com/gfx-rs/wgpu/pull/4935#discussion_r1436659586.

So my suggestion is not to remove the variants, it's to remove the "Available on `send_sync` only." documentation. Which isn't correct, it's available on both, just not with the `Send` bound on Wasm.

Reply:
ah, makes more sense 😄

Reply:
Done.